src/views/v2/day/event/featured-image.php

```diff
@@ -9,30 +9,28 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.4
+ * @version TBD
  *
  */
-$event    = $this->get( 'event' );
-$event_id = $event->ID;
 
-if ( ! has_post_thumbnail( $event_id ) ) {
+if ( empty( $event->thumbnail->large->url ) ) {
```

Comment:
`large` is a default size that could, but, be disabled by themes.
The only "safe", although inefficient, approach here is to use `full` that will always be available.